src/BatchEventData.ts

```diff
@@ -12,7 +12,7 @@ export enum TypedEventAttributeType {
   String = 's',
```

Comment:
Sorry for not adding this to each line. Please change
- s => string
- b => boolean
- i => integer

Reply:
makes sense! done